GVRf/Framework/src/org/gearvrf/GVRPicker.java

```diff
@@ -336,12 +336,12 @@ public static final float pickSceneObject(GVRSceneObject sceneObject,
      * @since 1.6.6
      */
     public static class GVRPickedObject {
-        private final GVRSceneObject sceneObject;
-        private final float[] hitLocation;
+        private final GVRSceneObject mHitObject;
+        private final float[] mHitLocation;
 
-        private GVRPickedObject(GVREyePointeeHolder holder) {
-            sceneObject = holder.getOwnerObject();
-            hitLocation = holder.getHit();
+        public GVRPickedObject(GVRSceneObject hitObject, float[] hitLocation) {
```

Comment:
this went from private to public.  i'm assuming there's a good reason?  and if so, all public methods of all public classes need javadoc.